ui/plugins/controller.js

```diff
@@ -88,7 +93,11 @@ treeherder.controller('PluginCtrl', [
                     $scope.eta_abs = Math.abs($scope.job.get_current_eta());
                     $scope.typical_eta = $scope.job.get_typical_eta();
                     $scope.jobRevision = ThResultSetStore.getSelectedJob($scope.repoName).job.revision
-
+                    $scope.job_ids = results[4];
+                    console.log(results);
+                    _.forEach($scope.job_ids, function(job) {
+                        console.log(job[0].value);
+                    });
```

Comment:
What is the job id and project name you're seeing this on? It's hard for me to diagnose what's going on without that information.